aws/logs_monitoring/trace_forwarder/cmd/trace/main.go

```diff
@@ -145,9 +145,16 @@ func sendTracesToIntake(tracePayloads []*pb.TracePayload) error {
 			fmt.Printf("Failed to send traces with error %v\n", err)
 			hadErr = true
 		}
+		stats := apm.ComputeAPMStats(tracePayload)
+		err = edgeConnection.SendStats(context.Background(), stats, 3)
+		if err != nil {
+			fmt.Printf("Failed to send trace stats with error %v\n", err)
```

Comment:
👀 you may have forgotten this?

Reply:
Going to leave this as `fmt` for now since it's consistent with rest of the file